### ISSUE TITLE
kfctl docker image - issue

### ISSUE THREAD

**Issue by micemuni** [kubeflow/kfctl#331]
Hello,

I have used https://github.com/kubeflow/kfctl/tree/v1.0.2 branch to build docker image, created kubernetes_job resource to run and mounted the kfctl_aws.yaml as config file, then container CMD is have "kfctl apply -V -f ${CONFIG_FILE}". 
I could see the container is created but re-cycling keep on and logs are showing 

[root@a-3b7daj8kxugj2 /kubeflow]# kubectl logs pod/kubeflow-767kc
standard_init_linux.go:211: exec user process caused "no such file or directory"
[root@a-3b7daj8kxugj2 /kubeflow]# 

```

Events:
  Type    Reason     Age   From                                   Message
  ----    ------     ----  ----                                   -------
  Normal  Scheduled  82s   default-scheduler                      Successfully assigned default/kubeflow-767kc to ip-xxxxx.ec2.internal
  Normal  Pulling    82s   kubelet, ip-xxxxx.ec2.internal pulling image "xxxxxxx.dkr.ecr.us-east-1.amazonaws.com/kubeflow/kfctl:969af6e"
  Normal  Pulled     30s   kubelet, ip-xxxxx.ec2.internal Successfully pulled image "xxxxxxx.dkr.ecr.us-east-1.amazonaws.com/kubeflow/kfctl:969af6e"
  Normal  Created    26s   kubelet, ip-xxxxx.ec2.internal  Created container
  Normal  Started    26s   kubelet, ip-xxxxx.ec2.internal  Started container

ENV vars:- 

    PATH             = "$PATH:/usr/local/bin/kfctl"
    CONFIG_URI       = "https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_aws.v1.0.2.yaml"
    AWS_CLUSTER_NAME = "kubeflow-us1"
    KF_NAME          = "kubeflow-us1"
    BASE_DIR         = "/opt/kubeflow"
    KF_DIR           = "/opt/kubeflow/kubeflow-us1"
    CONFIG_FILE      =  "/opt/kubeflow/kubeflow-us1/kfctl_aws.yaml"
```

and CMD is --

                  - container {
                      - args                     = [] 
                      - command                  = [
                          - "kfctl",
                          - "apply",
                          - "-V",
                          - "-f",
                          - "/opt/kubeflow/kubeflow-us1/kfctl_aws.yaml",
                        ] 

Please let me know if any mis-config or wrong.?

thank you

**Comment by shawnzhu** [comment]
See https://github.com/kubeflow/kfctl/issues/333#issuecomment-636588908

@micemuni I had a simple fix in #341. Do you want to give it a try?

**Comment by micemuni** [comment]
Hi,
I have fixed with change -- CGO_ENABLED=0 in makefile & added ENTRYPOINT ["/usr/local/bin/kfctl"] in Dockerfile.

makefile:
https://github.com/kubeflow/kfctl/blob/master/Makefile#L131
GOOS=linux GOARCH=amd64 CGO_ENABLED=0 ${GO} build -gcflags '-N -l' -ldflags "-X main.VERSION=$(TAG)" -o bin/linux/kfctl cmd/kfctl/main.go

Then build / deploy, now kfctl is working inside kfctl image
